htdocs/themes/xbootstrap/modules/lexikon/lx_index.tpl

```diff
@@ -200,18 +200,18 @@
     <div class="col-md-4 col-sm-12">
         <h3><{$smarty.const._MD_LEXIKON_RANDOMTERM}></h3>
         <{if $multicats == 1}>
-            <{if $empty != 1}>
+            <{if $empty|default:false != 1}>
                 <div class="catname"><a
                             href="<{$xoops_url}>/modules/<{$lang_moduledirname}>/category.php?categoryID=<{$random.categoryID}>"><{$random.categoryname}></a>
                 </div>
             <{/if}>
         <{/if}>
         <div class="pad4">
-            <h5 class="term"><{$microlinks}><a
-                        href="<{$xoops_url}>/modules/<{$lang_moduledirname}>/entry.php?entryID=<{$random.id}>"><{$random.term}></a>
+            <h5 class="term"><{$microlinks|default:null}><a
+                        href="<{$xoops_url}>/modules/<{$lang_moduledirname}>/entry.php?entryID=<{$random.id|default:null}>"><{$random.term|default:null}></a>
```

Comment:
May want to do a test here to see if any of these terms are not set correctly and 'skip' if any of them are 'null'.